docs/user-guide/cli-using-editing-team-configuration.md

```diff
@@ -41,3 +41,348 @@ To define additional mainframe services and other profiles in an existing global
         },
     }
     ```
+
+## Available service profile types
+
+A profile contains all, or most, of the information you need to connect to a specific mainframe service. Your configuration can have multiple profiles, and these can consist of different profile *types* and even different *kinds* of a particular profile type, depending on the connection information.
+
+There are three basic profile types:
+
+- service profiles
+- base profiles
+- parent profiles
+
+You can learn more about how service, base, and parent profiles work in [Zowe CLI profile types](../user-guide/cli-using-using-team-profiles.md#zowe-cli-profile-types).
+
+### Core z/OS service profiles
+
+The three z/OS services that Zowe CLI and Zowe Explorer profiles connect to:
+
+- **z/OSMF** profiles connect with the IBM z/OS Management Facility service.
+- **TSO** profiles connect with the Time Sharing Option service.
+- **SSH** profiles connect with the Secure Shell service.
+
+### Zowe CLI plug-in service profiles
+
+Other kinds of *service profiles* can be used to configure connections for Zowe CLI plug-ins. A *base profile*, on the other hand, contains connection data that can be shared across multiple service profiles.
+
+To determine the types of plug-in service profiles that can be used in Zowe CLI configuration, refer to the tables on this page or check the Zowe CLI plug-in command groups listed in the [Zowe web help](https://docs.zowe.org/stable/web_help/index.html). Most group names match the plug-in profile name.
+
+## Profile properties
+
+Every profile in a configuration file includes specific information, such as properties and their values, to communicate with its respective mainframe service. The values for properties are defined by your specific connection information.
+
+The available properties for z/OS services and Zowe-conformant plug-ins profiles are listed in the following tables:
+
+### base
+
+| Property | Description | Allowed |
+| --- | --- | --- |
+| host | Host name of service on the mainframe. | string |
+| port | Port number of service on the mainframe. | number |
+| user | User name to authenticate to service on the mainframe. | string |
+| password | Password to authenticate to service on the mainframe. | string |
+| rejectUnauthorized | Reject self-signed certificates.<br/><br/>**Default**: ```true``` | **boolean**:<br/> ```true```<br/>```false``` |
+| tokenType | The type of token to get and use for the API. Omit this option to use the default token type, which is provided by `zowe auth login`. | string |
+| tokenValue | The value of the token to pass to the API. | string |
+| certFile | The file path to a certificate file to use for authentication. <br/>  <br/> Note: <br/> The CLI does not support certificate files that require a password. For more information, see [PEM certificate files](../troubleshoot/cli/troubleshoot-cli-credentials.md#pem-certificate-files). | string |
```

Comment:
Please consider opening an issue with the corresponding plug-in (or CLI in this case) for every update that you had to make manually.
(unless you'd want to have the script enhanced) 🙏